docs/static-file-caching.md

```diff
@@ -0,0 +1,13 @@
+# Static File Caching
+
+All static files on Altis Cloud are cached on the CDN for a long period of time. This ensures the best performance when downloading static assets such as images, JavaScript and CSS files. Due to the aggressive caching times, when files are changed it is necessary reference the files by a new URL that will cause a different version to be saved on the CDN.
+
+Because query strings are part of the cache key when the file is cached on the CDN, in many cases it's simplest to reference the file with a changed `?version` query parameter. This behavior is automatic when providing the `$version` parameter to the `wp_enqueue_script` and `wp_enqueue_style` functions.
+
+```php
+wp_enqueue_script( 'my-theme', get_stylesheet_uri(), [], '2019-05-4-1' );
+```
+
+This will enqueue and output a `<script>` tag with a URL appended with `?ver=2019-05-4-1`. Every time the `style.css` is changed, the `$version` parameter needs to be updated.
```

Comment:
Could mention the deployment revision constant too? Currently `HM_DEPLOYMENT_REVISION`.

Reply:
Yeah I was going to, but then realized we don't have a better altis name for it. I think we should have another helper function ala `Altis\get_environment_architecture()` rather than the constant. Created https://github.com/humanmade/altis-core/issues/34 to track